src/index.tsx

```diff
@@ -15,13 +15,13 @@ interface App {
 }
 
 const smartLaunch = () => {
-
   FHIR.oauth2
     .init({
-      clientId: '20560ea5-f224-4658-b667-4e6bab935c85',
-      scope: 'launch/patient openid profile'
+      clientId: 'app-login',
```

Comment:
we might want to consider a solution similar to what DTR has, allowing registration of different client IDs for different fhir servers, to avoid more problems with hard coding it in.  It could also be an environment variable.

Reply:
Ariel has a task to make these configurable. I figured I'd just set them to what I needed for now.

Reply:
okay, sounds good.